src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/PrisonService.kt

```diff
@@ -45,4 +46,41 @@ class PrisonService(@Autowired private val client: PrisonApiClient, val faker: F
       reasonForAbsence = faker.expression("reason")
     )
   }.take((5..20).random()).toList()
+
+  @PreAuthorize("hasRole('BOOKING_CREATE') and hasAuthority('SCOPE_write')")
+  fun createAndAdmitOffender(createAndAdmitOffenderDetail: CreateAndAdmitOffenderDetail): CreateOffenderResponse {
+    val createResponse = client.createOffender(
```

Comment:
- The second api call below expects the accompanying OAuth token to contain a username that is known to NOMIS. If this name is absent the call will fail. Perhaps this method should check that the user invoking this method is known to NOMIS before the first call is made. 
- It would be possible for this service to lookup firstName, lastName and dateOfBirth from the BASM API if the method was passed the UUID of the BASM movement associated with this call.  However the invoker would still have to supply gender, movementReasonCode and imprisonmentStatus.
- The prisonId (agencyId) might also be determined as the invoking user's current caseload. If so then prisonId can be removed from the method signature.
- If this tier has exclude from the home page query BASM movements that resulted in successful admissions then this method must be give the UUID of the BASM movement.